django_auth_lti/static/django_auth_lti/js/resource_link_id.js

```diff
@@ -0,0 +1,12 @@
+window.globals.append_resource_link_id = function(url){
+    if (!url.match(/resource_link_id/)) {
+        if (!url.match(/\?/g)) {
+            url += '?';
+        }
+        return url + '&resource_link_id=' + window.globals.RESOURCE_LINK_ID;
```

Comment:
I'd like to avoid the "?&" in the url if we can...can we do something like this instead?

    var url_separator = (url.match(/\?/)) ? '&' : '?';
    return url + url_separator + 'resource_link_id=' + window.globals.RESOURCE_LINK_ID;

Also, was there a specific reason for the /g modifier on the regex?